Makefile

```diff
@@ -362,7 +362,7 @@ test: unit-test e2e-test
 
 .PHONY: unit-test
 unit-test: envtest
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $(TEST_SRC) -v  -coverprofile cover.out
+	OPERATOR_NAMESPACE=$(OPERATOR_NAMESPACE) KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $(TEST_SRC) -v  -coverprofile cover.out
```

Comment:
Unrelated change

Reply:
in fact it is not unrelated because as today, there is this code https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/pkg/cluster/cluster_config.go#L156-L161 with a `fmt.Printf`. Now that we can derive the openshift namespace from an env var, then we can set it when running the test and get rid of it.

Reply:
I can remove it and just swallow the error if you think it is better. Not sure what is the value of the printf

Reply:
> I can remove it and just swallow the error if you think it is better. Not sure what is the value of the printf

Yes, it can come with a separate PR like a followup to that patch which introduced environment variable and changed `make run` target.

Thanks!

Reply:
done

Reply:
https://github.com/opendatahub-io/opendatahub-operator/pull/1204 to follow